### DISCUSSION TITLE
Copying file at root

### DISCUSSION THREAD

**Discussion by fishingpanda** [vercel/community#2149] [Help] [answered]
### Summary

I am deploying a react.js application and I have a JSON file at root - which is not getting copied over. I am unable to figure out if there is a way to copy that file? appreciate the input.

**Comment by amyegan** [comment, marked as answer]
Hi @fishingpanda! If you're using create-react-app, assets should go in the `/public` folder to be picked up by React: https://create-react-app.dev/docs/using-the-public-folder/
